### PR TITLE
Increase max allowed  in UsageDetails

### PR DESCRIPTION
**Pull request by Panda-Wang**
## Description

Swagger has been updated with a bigger max# of $top thus update SDK with the same. Link to swagger RP: https://github.com/Azure/azure-rest-api-specs/pull/1165
Swagger: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-consumption/2017-04-24-preview/swagger/consumption.json
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [X] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
can you link to the swagger update.
Also was this generated using autorest, seems like you edited this.

**Comment by Panda-Wang** [comment]
Link to swagger RP: Azure/azure-rest-api-specs#1165
Swagger: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-consumption/2017-04-24-preview/swagger/consumption.json

The change was generated by AutoRest. There's another file change which I pushed as the 2nd commit. I didnt include it in 1st push because it's just comment update.

**Comment by shahabhijeet** [comment]
@Panda-Wang the reason I say that is because, I do not see generate.cmd using the commit id for the updated swagger that was checked in.

File: src/SDKs/Consumption/Management.Consumption/Generated/ConsumptionManagementClient.cs [hunk 1]
```diff
@@ -1,10 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
+// MICROSOFT_MIT┬á
```
Comment (shahabhijeet) [inline src/SDKs/Consumption/Management.Consumption/Generated/ConsumptionManagementClient.cs]: @Panda-Wang missing license header information

File: src/SDKs/Consumption/Management.Consumption/Generated/IConsumptionManagementClient.cs [hunk 2]
```diff
@@ -1,10 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
+// MICROSOFT_MIT┬á
```
Comment (shahabhijeet) [inline src/SDKs/Consumption/Management.Consumption/Generated/IConsumptionManagementClient.cs]: @Panda-Wang missing license header information

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Can you add the license header information.
We are fixing AutoRest to add that information back.

**Comment by shahabhijeet** [comment]
@Panda-Wang looks good. As soon as CI pass this PR can be merged.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**